base/util/src/main/java/com/netscape/cmsutil/crypto/CryptoUtil.java

```diff
@@ -2287,12 +2371,21 @@ public static SymmetricKey createDes3SessionKeyOnInternal() throws Exception {
         PublicKey pub = wrappingCert.getPublicKey();
         PK11PubKey pubK = PK11PubKey.fromSPKI(pub.getEncoded());
 
-        //Wrap the temp DES3 key with the cert
-        byte[] wrappedKey = wrapUsingPublicKey(token, pubK, wrappingKey, KeyWrapAlgorithm.RSA);
+        //Wrap the temp AES key with the cert
+        byte[] wrappedKey = wrapUsingPublicKey(token, pubK, wrappingKey, useOAEPKeyWrap ? KeyWrapAlgorithm.RSA_OAEP: KeyWrapAlgorithm.RSA); 
+
+        for(int i = 0 ; i < wrappedKey.length ; i ++) {
+
+            System.out.println(wrappedKey[i] + "");
+            if( i % 8 == 0) {
+                System.out.println("\n");
+            }
```

Comment:
As mentioned on IRC/... you could use write out `getModulus()`'s value as well, to compare the `pubK` value to the `privK`'s value later.